profiles/dgarcia.md

```diff
@@ -0,0 +1,3 @@
+# Damián García
+
+![](https://github.com/dami-gg.png)
```

Comment:
@frontendherodk because you have write access to this repository as a collaborator. GitHub's philosophy is very trust based. So we generally assume that people do not intentionally misuse their privileges.